app/(auth)/signup.js

```diff
@@ -62,6 +66,17 @@ export default function SignUp() {
         uniqueDrills: [],
       });
 
+      sendEmailVerification(auth.currentUser)
+        .then(() => {
```

Comment:
I think that we should be consistent with other code by using await here

Reply:
Should be resolved (moved to try / await syntax)